src/ProcessManager/ChromeManager.php

```diff
@@ -66,6 +66,10 @@ public function quit(): void
 
     private function findChromeDriverBinary(): string
     {
+        if (false !== $binary = \getenv('PANTHERE_CHROME_DRIVER_BINARY')) {
```

Comment:
we use `$_ENV` in other places (`getenv` isn't thread safe).

Reply:
Fixed